PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,19 @@
+# Goal
+
+[JIRA Ticket](https://jira)
+
+## What's Here
```

Comment:
We need to say that:

1. a PR without tests will be automagically r- (except some extremely rare scenarios).
2. the test code coverage cannot go down because of this PR. Also this is a good reason for a r-.
3. the PR must be green before ask for a review.